nilearn/surface/__init__.py

```diff
@@ -19,8 +18,14 @@
 )
 
 __all__ = [
+    "FileMesh",
+    "InMemoryMesh",
     "Mesh",
+    "PolyData",
+    "PolyMesh",
     "Surface",
+    "SurfaceImage",
+    "SurfaceMesh",
```

Comment:
question is if we want all of those to be part of the public API.

Reply:
They were already. What could be the issue ?

Reply:
I might be wrong, but I think for all practical purposes, we just need `SurfaceImage` in the public API.